### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**

### CODE REVIEW

File: rfm_examples/XBDP_SimpleTx.m [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+tx.NCOEnables = ones(1,numel(tx.EnabledChannels));
+tx.ChannelNCOGainScales = ones(1,numel(tx.EnabledChannels)).*0.5; %MxFE0 Digital Gain Code
+
+%DDS Configuration
+tx.DataSource = 'DDS'; %'DMA' or 'DDS'
+tx.EnableCyclicBuffers = 0; %0: Don't Cycle Tx Waveform, 1: Cycle Tx Waveform
+
+%DMA Configuration
+tx.DataSource = 'DMA'; %'DMA' or 'DDS'
+tx.EnableCyclicBuffers = 1; %0: Don't Cycle Tx Waveform, 1: Cycle Tx Waveform
```
Comment (tfcollins) [inline rfm_examples/XBDP_SimpleTx.m]: Why does this go back and forth?

File: rfm_examples/XBDP_TDD_Example.m [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+interFreq = 4.5e9; %Tx NCO Frequency & Unfolded Rx NCO Frequency [Hz]
+
+fs_RxTxIQ = 250e6; %IQ Sample Rate [Hz]
+periods = 32; %Desired Number Of Periods For Tx Signal
+basebandFreq = fs_RxTxIQ/periods; %Baseband Frequency [Hz]
+
+%Setup AD9081 Tx & Rx
+tx = adi.AD9081.Tx;
+rx = adi.AD9081.Rx;
+tx.uri = uri;
```
Comment (tfcollins) [inline rfm_examples/XBDP_TDD_Example.m:44]: RX is not used in this script. Is there something missing?

**Review by tfcollins — CHANGES_REQUESTED** [review]

File: rfm_examples/XBDP_SimpleTx.m (hunk 1 above)
Comment (SrikanthPagadarai) [reply rfm_examples/XBDP_SimpleTx.m]: Updated the file.

File: rfm_examples/XBDP_TDD_Example.m (hunk 2 above)
Comment (SrikanthPagadarai) [reply rfm_examples/XBDP_TDD_Example.m:44]: Updated the file.

**Review by tfcollins — APPROVED** [review]

**Pull request merged**